workflows/utilities/wf_flye_denovo.wdl

```
@@ -133,14 +133,14 @@ workflow flye_denovo {
   if (defined(illumina_read1) && defined(illumina_read2)) {
    call task_bwa_all.bwa_all as bwa {
      input:
-       draft_assembly_fasta = flye.assembly_fasta,
+       draft_assembly_fasta = select_first([flye.assembly_fasta]),
```

Are the additions of singular select_firsts intentional? These outputs are required in Flye so should throw a correct error if missing.

---

yeah, because they are now optional outputs of `task_flye.wdl` they must be `select_first`